Plugins/iOS/upload-dsym.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/sh
```

Comment:
That's interesting! We should document this, I guess. We can have symbolication for iOS. Actually, I'm not even sure what that helps, because it's all in C# Mono